spec/validators/birthday_validator_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+class Validatable
+  include ActiveModel::Validations
+  attr_accessor :birthday
+
+  validates :birthday, birthday: true
+end
+
+describe BirthdayValidator do
+  subject(:validatable) { Validatable.new }
+
+  context "when date is before current date" do
+    before { validatable.birthday = Time.zone.today - 1 }
+
+    it "is valid" do
+      expect(validatable).to be_valid
+    end
+  end
+
+  context "when date is equal current date" do
+    before { validatable.birthday = Time.zone.today }
+
+    it "is valid" do
+      expect(validatable).to be_valid
+    end
+  end
+
+  context "when date is greater than current date" do
+    before { validatable.birthday = Time.zone.today + 1 }
+
+    it "is invalid" do
```

Comment:
Nos casos desses três it "is valid" e "is invalid" não tem necessidade, tenta rodar pra ver como fica a documentação sem as descrições, pode por `it do`, o RSpec gera sozinho a documentação nesses casos e muitas vezes ja fazem sentido sozinhos

Reply:
Pode até colocar tudo isso inline `it { is_expected.to be_valid }` e `it { is_expected.to be_invalid }` vai ficar xou já! :nail_care:

Reply:
Resolvido, alterei conforme você orientou